tech/languages/csharp/dotnet-installation.md

```diff
@@ -8,26 +8,31 @@ order: 3
 
 ## Fedora .NET SIG repository
 
-.NET Core is still work in progress. The download location below contains packages that do not meet the official Fedora requirements yet (such as containing prebuilt binaries or bundled dependencies.) Please be patient, we should have reasonable packages within a few months.
+.NET Core is still work in progress. The download location below contains packages that do not meet the official Fedora requirements yet (such as using prebuilt tools to build the source) but are open source. Please be patient, we should have reasonable packages within a few months.
```

Comment:
Please remove the last sentence, as noone knows the time when this page was created without looking at the source and thus the information is pointless, or worse, may get obsolete.